pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -111,7 +111,7 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 	return lifecycle.PodAdmitResult{
 		Admit:   false,
 		Reason:  reason,
-		Message: message,
+		Message: getMessage(api.ResourceMemory),
```

Comment:
I changed this to use "disk" as default and use "memory" if the node condition fits. Let me know if you prefer checking each condition in a `switch` block, and defaults to a generic "resources" string. I don't think it matters now, so I went with the shorter (least) change.